libcxx/test/std/input.output/iostream.format/std.manip/setfill_wchar_max.pass.cpp

```diff
@@ -0,0 +1,38 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Test that WCHAR_MAX as a wchar_t value can be set as the fill character.
+
+// UNSUPPORTED: no-wide-characters
+
+// Expect the test case to fail on targets where WEOF is the same as
+// WCHAR_MAX with the libcpp ABI version 1 implementation. The libcpp ABI
+// version 2 implementation fixes the problem.
+
+// XFAIL: target={{.*}}-windows{{.*}} && libcpp-abi-version=1
+// XFAIL: target=armv{{71|81}}-linux-gnueabihf && libcpp-abi-version=1
```

Comment:
@xingxue-ibm I think there's a typo here (i.e. `l` for little endian not `1`) :
```suggestion
// XFAIL: target=armv{{7|8}}l-linux-gnueabihf && libcpp-abi-version=1
```

Reply:
Ah, good catch, thanks!